File: backend-springboot/src/main/java/com/javeriana/pontimovil/ponti_movil/dto/gestion_rutas/ruta_enviada/rEstacionEnviadaDTO.java
package com.javeriana.pontimovil.ponti_movil.dto.gestion_rutas.ruta_enviada;

import com.javeriana.pontimovil.ponti_movil.dto.gestion_rutas.estacion.rDireccionDTO;
import lombok.*;

import java.io.Serializable;
import java.util.UUID;

@Getter
@Setter
@AllArgsConstructor
@EqualsAndHashCode
@NoArgsConstructor
public class rEstacionEnviadaDTO implements Serializable {
    UUID id;
    String nombre;
    Integer orden;
    rDireccionDTO direccion;
}

File: frontend-angular/src/app/dto/gestionar-rutas/ruta-enviada/r-estacion-enviada-dto.ts
import {RDireccionDTO} from '../estacion/r-direccion-dto';

export class REstacionEnviadaDto {
  constructor(
    public id: number,
    public nombre: string,
    public  orden: string,
    public direccion: RDireccionDTO,
    public dentroRuta: boolean
  ) {}
}
